key/keys_test.go

```diff
@@ -139,18 +146,20 @@ func BatchIdentities(n int) ([]*Pair, *Group) {
 	for i := 0; i < n; i++ {
 		port := strconv.Itoa(startPort + i)
 		addr := startAddr + port
-		privs[i] = NewTLSKeyPair(addr)
+		privs[i], _ = NewTLSKeyPair(addr, nil)
 		pubs[i] = &Node{
 			Index:    uint32(i),
 			Identity: privs[i].Public,
 		}
 	}
-	fakeDistKey := KeyGroup.Point().Pick(random.New())
-	distKey := &DistPublic{[]kyber.Point{fakeDistKey}}
+	sch, _ := crypto.GetSchemeFromEnv()
```

Comment:
could move this above and use the scheme creating the `NewTLSKeyPair` - if it's not really needed in `NewTLSKeyPair` that could suggest it needs refactored out